src/client/sandbox/event/simulator.ts

```diff
@@ -32,6 +32,21 @@ const FOCUS_IN_OUT_EVENT_NAME_RE = /^focus(in|out)$/;
 // NOTE: initTextEvent method required INPUT_METHOD param in IE
 const DOM_INPUT_METHOD_KEYBOARD = 1;
 
+const mouseToPointerEventTypeMap = {
```

Comment:
Rename them in upper case notation.
`MOUSE_TO_POINTER_EVENT_TYPE_MAP`, `TOUCH_TO_POINTER_EVENT_TYPE_MAP`